src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,26 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  width: getCalendarWidth($cell-size, $gap-size);
+  gap: $gap-size;
+  padding: $calendar-padding;
+
+  &__item {
+    position: relative;
+    min-width: $cell-size;
+    height: $cell-size;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid #000;
+    margin-bottom: 1px;
```

Comment:
```suggestion

```
you gave gap and margin to item and now you have 1px gap between columns and 2px gap between rows